docker/entrypoint.sh

```diff
@@ -7,6 +7,7 @@ rm -f /app/tmp/pids/server.pid
 cd /app
 bundle config --global set build.sassc --disable-march-tune-native
 bundle install
-npm install
+yarn install
```

Comment:
To get this working locally, you need to rebuild the docker containers with no cache. Or else it will continue to use `npm install` on entry. Hopefully that wont be an issue when building containers for deployment